tests/src/FunctionalJavascript/ThunderJavascriptTestBase.php

```diff
@@ -72,13 +76,10 @@ protected function setUp() {
       $this->setGenerateMode(strtolower($_SERVER['generateMode']) === 'true');
     }
 
-    if (($facebook_id = getenv('FACEBOOK_APP_ID')) && ($facebook_secret = getenv('FACEBOOK_APP_SECRET'))) {
-      $instagram = \Drupal::configFactory()->getEditable('media_entity_instagram.settings');
-      $instagram->set('facebook_app_id', $facebook_id)
-        ->set('facebook_app_secret', $facebook_secret)
-        ->save();
-    }
-
+    $instagram = \Drupal::configFactory()->getEditable('media_entity_instagram.settings');
+    $instagram->set('facebook_app_id', 123)
+      ->set('facebook_app_secret', 123)
+      ->save();
```

Comment:
We don't need valid credentials anymore, because all requests are mocked.

Reply:
❤️